pkg/pillar/types/global.go

```diff
@@ -763,6 +765,7 @@ func NewConfigItemSpecMap() ConfigItemSpecMap {
 	configItemSpecMap.AddIntItem(ForceFallbackCounter, 0, 0, 0xFFFFFFFF)
 	configItemSpecMap.AddIntItem(EveMemoryLimitInBytes, uint32(eveMemoryLimitInBytes),
 		uint32(eveMemoryLimitInBytes), 0xFFFFFFFF)
+	configItemSpecMap.AddIntItem(VmmMemoryLimitInMiB, 0, 0, 0xFFFFFFFF)
```

Comment:
That maximum is a lot of MBytes! How about picking something reasonable like 1Pbyte or less i.e., the max value is 1024*1024?

Reply:
ok, will limit to 1024*1024*1024 MiB (1 PiB)

Reply:
done && PR updated